roles/endpoints/defaults/main.yml

```diff
@@ -226,3 +226,18 @@ endpoints:
       balance: 'roundrobin'
       prefer_primary_backend: False
       maxconn: 4096
+  aodh:
+    name: aodh
+    type: alarming
+    description: 'Alarming Service'
+    url:
+      admin: https://{{ fqdn }}:8042
+      internal: https://{{ fqdn }}:8042
+      public: https://{{ fqdn }}:8042
+    port:
+      backend_api: 8043
+      haproxy_api: 8042
```

Comment:
Will need to get this front-end port (8042)added to our firewall rules (vyatta).